proto/spire/api/server/trustdomain/v1/trustdomain.proto

```diff
@@ -73,9 +77,13 @@ message GetFederationRelationshipRequest {
     // (e.g., "example.org").
     string trust_domain = 1;
 
+    // If true, it returns the static federation relationship defined in the
+    // SPIRE server configuration file.
+    bool include_static = 2;
+
     // An output mask indicating which federation relationship fields
     // are set in the response.
-    spire.api.types.FederationRelationshipMask output_mask = 2;
+    spire.api.types.FederationRelationshipMask output_mask = 3;
```

Comment:
Note for other reviewers. This API has not been published/implemented yet so there are no existing clients to break by renumbering the fields. I was just about to comment with a "NO, DON'T!" when I realized it myself :)